components/footer.php

```diff
@@ -4,5 +4,7 @@
         <i class="fab fa-linkedin"></i>
         <i class="fab fa-instagram"></i>
         <i class="fas fa-envelope"></i>
+        <hr>
+    <p>&copy; Angel Angelov <?php echo date('Y'); ?></p>
```

Comment:
Exactly what needs to be done. Only avoid making updates in a merge request, which is about another update. The branch/update/merge request is about `Improving-navigation` so this commit should not be there. For now its fine and will be merged as it is valid code.

Reply:
I see what happened let me correct myself. You have added this commit first in the branch `add-copyright-footer`. Then when you made your new branch `Improving-navigation` (this one) you created it from the `add-copyright-footer`. Since the copyright branch was not merged, this commit entered this request.

To avoid that do the following: Let's say you make this commit in the branch `add-copyright-footer` and create a merge request - so far so good. Next, you want to make the branch `Improving-navigation` by first checking out in master and then creating the `Improving-navigation` branch - in this way we isolate branches. It is important and good practice. If you have a hard time understanding me, it would be good to make a call and explain it again.